### PR TITLE
Update blog content

### PR DESCRIPTION
**Pull request by isisblack**

### CODE REVIEW

**Comment by pruess-google** [comment]
> ID should be a unique thing on the page.
> 
> generally its used for anchor tags so you can link to it by going to blog#blog-content
> 
> in this instance the browser would be confused since there are two.
> 
> In this case, you probably just want to use class when sharing css.

+1, this thread has some more info:
https://stackoverflow.com/q/544010

It looks like you're using the IDs for styling, so I think you can change them to classes and have everything still work correctly.

**Comment by isisblack** [comment]
Fixed!

File: portfolio/src/main/webapp/projects.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="gallery.html">Gallery</a></li>
+            <li><a href="blog.html">Blog</a></li>
+        </ul>
+    </nav>
+    <div class="heading">
+        <h1>My Projects</h1>
+    </div>
+   <div id="projects-content">
+    <div id="content">
```
Comment (pruess-google) [inline portfolio/src/main/webapp/projects.html:22]: I'm also seeing duplicate IDs in this file and in gallery.html. Would you mind updating them as well?

Comment (isisblack) [reply portfolio/src/main/webapp/projects.html]: @pruess-google Fixed those too!

**Pull request merged**